templates/email/default/_email_comment_list.html

```diff
@@ -5,7 +5,7 @@
         <img style="[% list_item_photo_style %]" src="[% inline_image(update.get_first_image_fp) %]" alt="">
       </a>
     [%~ END %]
-      [% email_sanitize_html(update) | html_para | replace('<p>', '<p style="' _ list_item_p_style _ '">') %]
+      [% email_sanitize_html(update) | replace('<p>', '<p style="' _ list_item_p_style _ '">') %]
```

Comment:
Why are some of these using the new filter and some not?

Reply:
turns out email_sanitize_html is already calling html_para, so I can't pass it to the new filter (which would call html_para again). So didn't seem worth doing anything about.